mappings/net/minecraft/fluid/FluidState.mapping

```diff
@@ -1,5 +1,5 @@
 CLASS net/minecraft/class_3610 net/minecraft/fluid/FluidState
-	METHOD method_15756 (Lnet/minecraft/class_1922;Lnet/minecraft/class_2338;)Z
+	METHOD method_15756 isVisibleFromASideClient (Lnet/minecraft/class_1922;Lnet/minecraft/class_2338;)Z
```

Comment:
What does "client" in indicate? Is this like a method to check culling?

Reply:
@SideOmly

Reply:
Definitely a bad name. We tend to avoid adding a `Client` to method name simply because it's annotated with `@Environment(EnvType.CLIENT)`

Reply:
Cc @pyrofab

Reply:
Yeah yeah, I just thought it was really convenient